packages/webapp/src/components/SensorReadingsLineChart/index.jsx

```diff
@@ -207,8 +227,16 @@ const PureSensorReadingsLineChart = ({
                   strokeWidth={2}
                   dataKey={attribute.value}
                   stroke={attribute.color}
-                  activeDot={{ r: 6 }}
                   isAnimationActive={false}
+                  activeDot={{ r: 5 }}
+                  dot={dataDensity(sensorReadings, attribute.value) < 0.9}
```

Comment:
@kathyavini I am not sure this acts as intended. With the following graph it is missing datapoints in one 24 hour period but the density for the others is still ~100%.

<img width="610" alt="Screenshot 2023-05-24 at 9 52 42 AM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/48864823-1152-453d-a6d8-f9faad8b1166">